varnish-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,936 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
```

Comment:
For panels with an increase function in the query, we set a min step of '1m' (find this under the query, 'Step: auto')

Reply:
All the ones with the increase do already have the 1m step. Did you see a specific one that did not?